docs/examples.md

```diff
@@ -301,3 +303,20 @@ Common configuration properties that you might want to change from their default
 
 - [cache.max.bytes.buffering](https://kafka.apache.org/documentation/#streamsconfigs):
   The default value in KSQL is `10000000` (~ 10 MB);
+
+
+## Running KSQL
+
+KSQL support various [modes of operation](concepts.md#modes-of-operation), including a standalone mode and a
+client-server mode.
+
+Additionally, you can also instruct KSQL to execute a single statement from the command line.
+
+Example:
```

Comment:
i'd move this text before the example, and consolidate a few things: 
`The following example runs a SELECT statement until 5 records have been found and then the results are shown in the terminal. `